### ISSUE TITLE
Kustomize directory analysis not working

### ISSUE THREAD

**Issue by eddylacrombe** [kubescape/kubescape#1617]
# Description
As described in the kubescape docs we can analyze a kustomize repository.
In my Kustomize directory I have in my base folder a webserver with resource limit set
![Capture d’écran 2024-02-27 à 08 39 14](https://github.com/kubescape/kubescape/assets/64368327/f74e4014-5b39-428c-af14-d86d24540d5a)

But when I'm running 
`kubescape scan --verbose framework nsa path-to-my-kustomize-dir/overlays/prod` I still get  a high severity for resource limits.
![Capture d’écran 2024-02-27 à 08 36 24](https://github.com/kubescape/kubescape/assets/64368327/d09659fc-d014-48fa-be0d-6897139d6c02)

I've tried running 
`kubescape scan --verbose framework nsa path-to-my-kustomize-dir/`
`kubescape scan --verbose framework nsa path-to-my-kustomize-dir/base` 
but it didn't work either

# Environment
OS: ` macOS Ventura 13.6`
Version: `v3.0.3 `
 
# Steps To Reproduce
1. Create a kustomize directory such as this one 
<img width="104" alt="Capture d’écran 2024-02-27 à 08 35 32" src="https://github.com/kubescape/kubescape/assets/64368327/3de62379-c2e8-4094-b5a2-8b0c97ebea53">
2. In the directory create a deployment as base with resources limits set at the container level
3. In one of the overlays directory create an kustomization of the deployment created at the previous step (not one setting resource limits obviously). In my case I'm changing the image of the container deployed in my pod.
4. Run `kubescape scan --verbose framework nsa path-to-my-kustomize-dir/overlays/prod`, you'll have a severity high on the resource limits.

# Expected behavior
I'd expect the resource limits severity not to be a failed resource and the template rendering issues for the complete deployment not only for the overlay

# Actual Behavior
The resource limits severity is considered a failed resource however it was correctly set

# Additional context

**Comment by bhavyastar** [comment]
Hey @matthyx 
I would like to work on this issue.

**Comment by VasuDevrani** [comment]
@bhavyastar still on it?

**Comment by bhavyastar** [comment]
Yes working on this issue. Got exams now from last 2 days I am working on this issue.

**Comment by mastersans** [comment]
@bhavyastar still on it?? If not i would like to work on it @matthyx !

**Comment by 102vansh** [comment]
i want to solve this issue
